src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -4096,6 +4219,31 @@ namespace InternalHeatGains {
 			}
 		}
 
+		std::map< int, std::vector< int > >::iterator it = ZoneITEMap.begin();
+		Real64 totalGain;
```

Comment:
@Myoldmopar You ok with using a map here?

Reply:
It could be cleaned up to become a regular for loop instead of initializing begin() out here and then writing the while loop, but it doesn't make a significant difference.  At this point I'm OK with it.